project/Settings.scala

```diff
@@ -0,0 +1,83 @@
+import sbt._
+
+/**
+ * Application settings and dependencies
+ */
+object Settings {
+  object Definitions {
+    /** The name of the application */
+    val name = "ocs3"
+
+    /** Project version */
+    val version = "2016001.1.1"
```

Comment:
We should put the version in its own SBT file (as done [here](https://github.com/tpolecat/atto) for instance) and yes, mark it as a snapshot. I'd like to get set up with sbt-release and all that eventually.